src/main/resources/templates/index.html

```diff
@@ -24,7 +24,16 @@
         });
     </script>
     <script th:src="@{/theme-toggle.js}"></script>
-
+    <script src="js/index.js"></script>
```

Comment:
Would it be possible to only load this at challenge16 using thymeleaf conditionals?

Reply:
Done